env/metaphysics.env

```diff
@@ -0,0 +1,43 @@
+# Service Dependencies
+######################################################################
+
+# AWS (localstack)
+AWS_ACCESS_KEY_ID=local
+AWS_BUCKET_PATH=metaphysics/heapsnapshots/staging
```

Comment:
Need to double check if AWS is only here because we previously uploaded our heap dumps to a bucket.

Reply:
I think so: https://github.com/artsy/metaphysics/search?q=aws_access_key_id